gbpservice/nfp/Automation-Scripts/local.conf

```diff
@@ -125,7 +125,7 @@ default_proxy_ip_pool = 192.169.0.0/16
 default_external_segment_name = default
 
 [device_lifecycle_drivers]
-drivers = haproxy, vyos
+drivers = haproxy, vyos, haproxy_lbaasv2
 
```

Comment:
These changes not needed now, as vendor specific drivers are removed from orchestrator